packages/cf-icons/src/cf-icons-svg-inline.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
 
     /**
      * @param {string} svgName - The canonical name of the icon.
-     * @param {string} svgFillColor - The fill color of the icon in rgb() format (defaults to black).
+     * @param {string} svgFillColor - The fill color of the icon (defaults to CFPB Black).
```

Comment:
We want to default to CFPB black for the icons? I'm not sure where CFPB versus pure black is used—but if it's an icon with black text it should be CFPB black?

Reply:
The only non-legacy places where we use pure black are in the video overlay (where we thought a not-pure-black background alongside a video could be distracting) and in print styles, where pure black is supposed to print faster.

Reply:
Ah interesting ok